mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/changelog/highlevel/OrientDBHighLevelChangesList.java

```diff
@@ -19,36 +16,30 @@
  *
  */
 @Singleton
-public class OrientDBHighLevelChangesList implements HighLevelChangesList {
+public class OrientDBHighLevelChangesList extends HighLevelChangesList {
 
 	@Inject
 	public RestructureWebrootIndex restructureWebroot;
 
 	@Inject
 	public FixNodeVersionOrder fixNodeVersionOrder;
 
-	@Inject
-	public ExtractPlainText plainText;
-
-	@Inject
-	public SetAdminUserFlag setAdminUserFlag;
-
 	@Inject
 	public OrientDBHighLevelChangesList() {
 	}
 
 	@Override
 	public List<HighLevelChange> getList() {
-		return Arrays.asList(
-			restructureWebroot,
-			plainText,
-			fixNodeVersionOrder,
-			setAdminUserFlag
+		List<HighLevelChange> changeList = super.getList();
+
 		// ADD NEW CHANGES HERE!
 		// WARNING!
 		// Only add changes when absolutely needed. Try to avoid changelog entries since those would require a offline update.
 		// Instead it is recommended to write changes in a way so that those can be applied on the fly.
 		// WARNING!
-		);
+		changeList.add(restructureWebroot);
```

Comment:
this will throw an UnsupportedOperationException, since the list was generated with Arrays.asList() in super.getList()